wgpu-core/src/device/mod.rs

```diff
@@ -342,7 +342,6 @@ impl<B: GfxBackend> Device<B> {
                         spv::Capability::Image1D,
                         spv::Capability::SampledCubeArray,
                         spv::Capability::ImageCubeArray,
-                        spv::Capability::ImageMSArray,
```

Comment:
this turns out to be multisampled storage textures, we don't need them (at least right now)